README-PT.md

```diff
@@ -0,0 +1,94 @@
+# IDP Guideline
+
+# Guia IDP (Plano de desenvolvimento pessoal) 📈
+
+![Curvas multiplcas do: IDP Guideline](https://res.cloudinary.com/daiqkausy/image/upload/v1594582756/idp-cover.png)
```

Comment:
The first part should be the image's description (_alt_), I think it would be better to be something like: "Caminho curvo com várias placas apontando em diversas direções e título: IDP Guideline — Individual Development Guide].

Maybe it would be nice to have an image with the texts in Portuguese 🙂

Reply:
@luistak hey, this image is yours?
Because if yes, I can change for Portuguese. I wasn't find any image equal it;